res/xml/preferences_advanced.xml

```diff
@@ -89,5 +89,11 @@
 
         <Preference android:key="pref_view_log"
                     android:title="@string/pref_view_log"/>
+
+        <org.thoughtcrime.securesms.components.SwitchPreferenceCompat
```

Comment:
can't we reuse the developer option for this? some extra debugging that makes the app slower and perhaps unreliable sounds like a perfect fit :D

Reply:
Hm, then we don't warn the user as explicitly about the slowndown, and we can't disable the option when the webxdc is deleted, so this "the webxdc is there exactly when the option is on" relationship will be gone.

And the developer option is for people who develop webxdcs (at least currently) while the debug logging option is for people who want to track down issues with deltachat core, which doesn't necessarily belong together.